R/select-tbl.R

```diff
@@ -72,8 +72,8 @@ filter_recode_table_fks <- function(def, selected) {
 }
 
 filter_recode_table <- function(data, selected) {
-  data %>%
-    filter(table %in% !!selected) %>%
+  reduced_selected <- tibble(table = selected[selected %in% data$table])
```

Comment:
Would that work:

```suggestion
  reduced_selected <- tibble(table = unname(selected))
```

We're checking inputs earlier on.

Reply:
produces an error in the tests. I think the problem is that `filter_recode_table()` is used in a few different places, among others in `cdm_rm_pk()`. I wouldn't want to dig too deep, cause it seems like a waste of time. If you want I can try to figure it out though.